source/gsoc.md

```diff
@@ -141,11 +141,9 @@ Once the ASF is confirmed as a mentoring organisation mentors must register
 with the GSoC webapp, and request to become a mentor for the ASF
 organization. Make sure that the email address you use for that (it's often
 your @gmail.com address by default) is '''registered as a mail alias for your Apache account at (https://id.apache.org)''' so that we can match it to your ASF account.
-You should also register your '''GSoC Link Id to
-committers/GSoCLinkId.txt'''.
 
 All ASF projects are invited to submit their ideas to their issue tracker,
-please be sure to add the labels "gsoc2021" and "mentor" so that we can
+please be sure to add the labels "*gsoc2021*" and "*mentor*" so that we can
 automatically include them in our list of subjects. If your project does
 not use JIRA please contact dev@community.apache.org.
```

Comment:
Hi @solomax One similar expression here. Please give it attention. Except for that, these guidelines make sense to me. Nice job. 😉

Reply:
I leave it copy/pasted due I'm unsure how to `reuse` the code
I'll merge this version if you don't have any better idea :)))